src/moj/components/date-picker/template.njk

```diff
@@ -0,0 +1,64 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/label/macro.njk" import govukLabel %}
+{% from "govuk/components/hint/macro.njk" import govukHint %}
+{% from "govuk/components/error-message/macro.njk" import govukErrorMessage %}
+
+<div class="govuk-form-group {%- if params.errorMessage %} govuk-form-group--error{% endif %} moj-datepicker {{ params.classes }}" data-module="moj-date-picker">
+  {{ govukLabel({
+    html: params.label.html,
+    text: params.label.text,
+    classes: params.label.classes,
+    isPageHeading: params.label.isPageHeading,
+    attributes: params.label.attributes,
+    for: params.id
+  }) | indent(2) | trim }}
+  {% if params.hint %}
+    {% set hintId = params.id + "-hint" %}
+    {% set describedBy = describedBy + " " + hintId if describedBy else hintId %}
+    {{ govukHint({
+      id: hintId,
+      classes: params.hint.classes,
+      attributes: params.hint.attributes,
+      html: params.hint.html,
+      text: params.hint.text
+    }) | indent(2) | trim }}
+  {% endif %}
+  {% if params.errorMessage %}
+    {% set errorId = params.id + "-error" %}
+    {% set describedBy = describedBy + " " + errorId if describedBy else errorId %}
+    {{ govukErrorMessage({
+      id: errorId,
+      classes: params.errorMessage.classes,
+      attributes: params.errorMessage.attributes,
+      html: params.errorMessage.html,
+      text: params.errorMessage.text,
+      visuallyHiddenText: params.errorMessage.visuallyHiddenText
+    }) | indent(2) | trim }}
+  {% endif %}
+  <div class="moj-datepicker-input__wrapper">
+    {{ govukInput({
+      classes: "govuk-input moj-js-datepicker-input",
+      id: params.id,
+      name: params.name,
+      value: params.value,
+      autocomplete: "off",
+      attributes: {
+        "maxlength": "10",
```

Comment:
GDS advise against creating hard-restrictions for users like this, it should be handled by post-submission validation instead